articles/aks/node-autoprovision.md

```diff
@@ -64,11 +64,23 @@ NAP is based on the Open Source [Karpenter](https://karpenter.sh) project, and t
     ```
 
 ## Limitations
-
-- Windows and Azure Linux node pools aren't supported yet
-- Kubelet configuration through Node pool configuration is not supported
-- NAP can only be enabled on new clusters currently
-- It is not currently possible to stop nodepools or clusters which use the NAP feature
+### Unsupported features:
+- Windows and Azure Linux node pools
+- Kubelet configuration through Node pool configuration
+- IPv6 clusters
+- [Service Principals](/azure/devops/integrate/get-started/authentication/service-principal-managed-identity)
```

Comment:
I think this should be linking to the following article - kubernetes-service-principal.md that is part of the AKS content set.